### DISCUSSION TITLE
Modal re-render isse

### DISCUSSION THREAD

**Discussion by Ampex** [heroui-inc/heroui#503] [Help]
Hello, I found an issue, that `Modal` on opened state is not re-rendering `{ children }` if they changed. My idea is to dynamically change a content of Modal body with context API, but now it is impossible... what do you think ?

**Comment by tianenpang** [comment]
Could you provide a reproduction? here's an example: [codesandbox.io/s/nextui-modal-update-4btxqf](https://codesandbox.io/s/nextui-modal-update-4btxqf?file=/pages/index.js)
